packages/shaderlab/src/shaders/shadingPBR/materialInputPBR.glsl

```diff
@@ -63,42 +66,6 @@ float material_OcclusionTextureCoord;
     sampler2D material_OcclusionTexture;
 #endif
 
-// Runtime
-struct ReflectedLight {
-    vec3 directDiffuse;
-    vec3 directSpecular;
-    vec3 indirectDiffuse;
-    vec3 indirectSpecular;
-}
-
-struct Geometry {
-    vec3  position;
-    vec3  normal;
-    vec3  viewDir;
-    float dotNV;
-    
-    #ifdef MATERIAL_ENABLE_CLEAR_COAT
-        vec3 clearCoatNormal;
-        float clearCoatDotNV;
-    #endif
 
-    #ifdef MATERIAL_ENABLE_ANISOTROPY
-        vec3  anisotropicT;
-        vec3  anisotropicB;
-        vec3  anisotropicN;
-        float anisotropy;
-    #endif
-}
-
-struct Material {
-    vec3  diffuseColor;
-    float roughness;
-    vec3  specularColor;
-    float opacity;
-    float f0;
-    #ifdef MATERIAL_ENABLE_CLEAR_COAT
-        float clearCoat;
-        float clearCoatRoughness;
-    #endif
 
-}
\ No newline at end of file
+#endif
```

Comment:
I find that your assumptions about the concept of Input are not clear

- input.glsl is attributes and varing;
- materialInputPBR.glsl is uniform;


I understand that Input itself is an abstract concept, but this is not good for user understanding and readability.

Reply:
done